frontend/src/metabase/dashboard/components/Dashboard/Dashboard.tsx

```diff
@@ -266,13 +243,14 @@ function DashboardInner(props: DashboardProps) {
     }
 
     const currentTabParameterIds = currentTabDashcards.flatMap(
-      dc => dc.parameter_mappings?.map(pm => pm.parameter_id) ?? [],
+      (dc: DashboardCard) =>
+        dc.parameter_mappings?.map(pm => pm.parameter_id) ?? [],
     );
     const hiddenParameters = parameters.filter(
-      parameter => !currentTabParameterIds.includes(parameter.id),
+      (parameter: Parameter) => !currentTabParameterIds.includes(parameter.id),
```

Comment:
```suggestion
      parameter => !currentTabParameterIds.includes(parameter.id),
```
hiddenParameters is already typed, any reason not to leave ts infer the type?